packages/apollo-language-server/src/project.ts

```diff
@@ -179,6 +200,38 @@ export class GraphQLProject {
     return !!this.setThatIncludes(filePath);
   }
 
+  async loadSchemaTags() {
+    await this.loadingHandler.handle(
+      `Loading available schema tags for ${this.config.name!}`,
+      Promise.all(
+        Object.values(this.config.schemas!).map(async schemaDef => {
+          if (schemaDef.engineKey) {
```

Comment:
should we not even go into loading if we don't have an engineKey for any service? And instead flag a missing value

Reply:
a8f44782656fd1993482baee82c58f280c3a11f8